specs/StreetLighting/Streetlight/doc/spec.md

```diff
@@ -75,6 +75,11 @@ The data model is defined as shown below:
     -   Allowed values: one Of (`on`, `off`, `low`, `bootingUp`)
     -   Optional
 
+-   `refDevice` : Reference to the device(s) used to monitor this streetligth.
+
```

Comment:
nit: new line not needed

Reply:
I readded this new line to follow the markdown style used by Jason in cc50f668989b04e7d7b3d745063d9c25ec3e23c7, as you can see, the other attributes are using this separation line (you can see for example the location attribute,  https://github.com/Fiware/dataModels/blame/master/specs/StreetLighting/Streetlight/doc/spec.md#L17).